apps/server-e2e/src/tests/csrf.ts

```diff
@@ -0,0 +1,53 @@
+import { parse } from 'lightcookie';
+import { spec } from 'pactum';
+import { Callback } from 'uvu';
+import { ok, unreachable } from 'uvu/assert';
+
+export const csrfTests: Callback<any> = async (_context: any) => {
+  await spec()
+    .get('/csrf')
+    .expectStatus(200)
+    .expectJsonLike({ csrfToken: /\w+/ })
+    .expect(({ res }) => {
+      const cookies = res.headers['set-cookie'];
+      if (!cookies || cookies.length === 0) {
+        unreachable('Received no cookies from the server');
+        return;
+      }
+      cookies.forEach((cookie) => {
+        const parsed = parse(cookie);
+        ok(
+          ['sessionId', 'refreshId'].some((cookieName) => {
+            return cookieName in parsed;
+          })
+        );
+      });
```

Comment:
Custom cookie parsing due to [a bug in lightcookie](https://github.com/pactumjs/pactum/issues/287) that is reported to pactum.